src/base/seeds/Seed.ts

```diff
@@ -100,12 +100,14 @@ export class Seed {
     return proj.Project.getInfo(urn, this.api);
   }
 
-  async getProjects(id?: string): Promise<proj.ProjectInfo[]> {
+  async getProjects(config: Config, id?: string): Promise<proj.Project[]> {
     const result = id
       ? await proj.Project.getDelegateProjects(id, this.api)
       : await proj.Project.getProjects(this.api);
 
-    return result.map((project: proj.ProjectInfo) => ({ ...project, id: project.urn }));
+    return Promise.all(
+      result.map((project: proj.ProjectInfo) => proj.Project.get(project.urn, null, null, this.api.host, config))
+    );
```

Comment:
I think it would be best to separate these api queries, otherwise we have to wait for lots of requests to complete before the page can render.

Reply:
Ok reverted this to just map the urn to `id`